doc/user/content/cli/_index.md

```diff
@@ -200,3 +202,61 @@ etc.), and then create a new node with those items.
 [gh-feature]: https://github.com/MaterializeInc/materialize/issues/new?labels=C-feature&template=feature.md
 [scv]: /sql/show-create-view
 [scs]: /sql/show-create-source
+
+### Persistence
+
+{{< warning >}}
+
+Persistence is a experimental feature at this time. Please don't use it
+with any data you are not comfortable losing. To use persistence you need to have
+both the `--persistence` and `--experimental` flags enabled.
+
+{{< /warning >}}
```

Comment:
Can you adjust this to use the experimental shortcode instead, for consistent wording? There's an example in `sql/create-table.md` that you can follow!